xilem/src/lib.rs

```diff
@@ -27,36 +29,32 @@ pub use xilem_core as core;
 
 mod any_view;
 pub use any_view::AnyWidgetView;
+
 mod driver;
+pub use driver::{async_action, MasonryDriver, MasonryProxy, ASYNC_MARKER_WIDGET};
+
 pub mod view;
 
-pub struct Xilem<State, Logic, View>
-where
-    View: WidgetView<State>,
-{
-    pub root_widget: RootWidget<View::Widget>,
-    pub driver: MasonryDriver<State, Logic, View, View::ViewState>,
+/// Re-export of tokio as the async driver for Masonry
+pub use tokio;
```

Comment:
I wonder how feasible it is to be generic over the async runtime (remembering a TODO comment in the classic archive). But it probably results in "overabstraction"...

Also is it "Masonry" or "Xilem" (which IMO would make more sense?)?

Reply:
There isn't a compelling ecosystem story about abstracting over this - you need to make a choice, and the default choice is certainly Tokio.

Thanks for catching the comment